PasswordValidator_UnitTests/AdvancedValidator_UnitTests.cs

```diff
@@ -26,7 +26,7 @@ public void IsValidAdvancedPassword_ValidPassword_ShouldReturnTrue(string advanc
         [TestCase("2300fmnenvieu", Description = "No Special Character")]
         public void IsValidAdvancedPassword_InvalidPassword_ShouldReturnFalse(string advancedPassword)
         {
-            IPasswordValidator validator = _factory.CreateValidator();
+            IPasswordValidator validator = _factory.GetPasswordValidator("Advanced");
 
             bool result = validator.Validate(advancedPassword);
 
```

Comment:
How would you test for an exception being thrown? Can you force it to throw an exception?